packages/quantic/force-app/examples/main/lwc/exampleQuanticCategoryFacet/exampleQuanticCategoryFacet.js

```diff
@@ -0,0 +1,77 @@
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticFacet extends LightningElement {
+    @api engineId = 'quantic-category-facet-engine';
+    @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Category Facet';
+    pageDescription = 'The Quantic Category Facetis a list of values for a certain field occurring in the results, ordered using a configurable criterion.'
```

Comment:
I think we should stress that the category facet is meant to be used with hierarchical values.

Reply:
```suggestion
    pageDescription = 'The Quantic Category Facet is a list of values for a certain field occurring in the results displayed in a browsable, hierarchical fashion and ordered using a configurable criterion.'
```